Readme.md

```diff
@@ -2,25 +2,25 @@
 
 [![Build Status](https://travis-ci.org/vurtun/nuklear.svg)](https://travis-ci.org/vurtun/nuklear)
 
-This is a minimal state immediate mode graphical user interface toolkit
+This is a minimal-state immediate-mode graphical user interface toolkit
```

Comment:
@sglass68: I don't think "minimal-state" is typically hyphenated (at least, in a little bit of googling I'm not finding examples of it that way)

Reply:
I'm not sure google is going to help much. The way I read it, it is creating an adjective out of a noun (state).

What is this sentence intended to mean? I assumed:

It is a GUI toolkiit
It uses immediate mode
It keeps minimal state internally

'This is a minimal state' probably references to a country ;-)

Perhaps I should have suggested: This is a GUI toolkit which keeps minimal state and operates in immediate mode?

Reply:
@sglass68:

> it is creating an adjective out of a noun (state)

You're right, good point.

> Perhaps I should have suggested: ...

I'm happy either way (though I slightly prefer the `minimal-state immediate-mode` wording, as it is currently in this PR).